examples/emrcontainers/virtualcluster.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: emrcontainers.aws.upbound.io/v1beta1
+kind: VirtualCluster
+metadata:
+  name: example
+spec:
+  forProvider:
+    containerProvider:
+      - info:
+        - eksInfo:
+          - namespace: emr-on-eks
```

Comment:
The uptest fails. Does this resource require an EKS cluster to be referenced here?

Reply:
Yes. The ID and namespace field tells which EKS cluster and namespace to connect EMR to.

Reply:
As you mentioned, this resource needs an EKS cluster to run. So, please add an EKS cluster manifest to this file like this:

```yaml
apiVersion: emrcontainers.aws.upbound.io/v1beta1
kind: VirtualCluster
...
...

---

apiVersion: eks.aws.upbound.io/v1beta1
kind: Cluster
...
...
```

And use this EKS cluster resource as reference in VirtualCluster resource:

```yaml
containerProvider:
  - idSelector:
      matchLabels:
        testing.upbound.io/example-name: example
```

Reply:
@nabuskey Do you need help with this PR?

Reply:
@sergenyalcin Sorry for the late reply. Adding EKS cluster may not be sufficient for this since it requires ns, role and role binding. Am I allowed to add these to the file as well?

Reply:
Sure, please add them. Thank you!